hudi-spark-datasource/hudi-spark-common/src/main/scala/org/apache/hudi/HoodieCLIUtils.scala

```diff
@@ -40,8 +40,12 @@ object HoodieCLIUtils extends ProvidesHoodieConfig{
       .setConf(sparkSession.sessionState.newHadoopConf()).build()
     val schemaUtil = new TableSchemaResolver(metaClient)
     val schemaStr = schemaUtil.getTableAvroSchema(false).toString
+    val table = metaClient.getTableConfig.getDatabaseName match {
+      case databaseName: String => databaseName + "." + metaClient.getTableConfig.getTableName
+      case _ => metaClient.getTableConfig.getTableName
+    }
     val finalParameters = HoodieWriterUtils.parametersWithWriteDefaults(
-      buildHoodieConfig(getHoodieCatalogTable(sparkSession, metaClient.getTableConfig.getTableName)) ++ conf)
+      buildHoodieConfig(getHoodieCatalogTable(sparkSession, table)) ++ conf)
```

Comment:
MOR Table's `rt` or `ro` table may not be found.

Reply:
@XuQianJin-Stars This method is only triggered by call procudure, user is not allowed to use ro/rt table

Reply:
> @XuQianJin-Stars This method is only triggered by call procudure, user is not allowed to use ro/rt table

call run_compaction(op => 'run', table => 'db.test_rt');

Reply:
@XuQianJin-Stars I returned the logic of constructing config before https://github.com/apache/hudi/pull/8243. And just add catalog props only when tableName is provided